packages/transport/scripts/protobuf-patches/TxOutputType.js

```diff
@@ -0,0 +1,50 @@
+// @flow
```

Comment:
how is possible that flow types works here?

Reply:
It does not. This annotation should be removed probably.

Reply:
Ok, it needs to stay there. Just for a while. Right now it is still possible to generate flowtype definitions using these files in trezor-connect (https://github.com/trezor/connect/pull/1077) removing // flow anotation would break it.

Reply:
Maybe add comment here with this info

Reply:
ok, will do. we are removing this part completely in maybe 2 weeks btw.

Reply:
uh, eh, no.. lets remove it some day later. I don't want to touch this now. will be really soon.